rag-langchain/rag_app.py

```diff
@@ -18,15 +18,22 @@
 import argparse
 import pathlib
 
+model_service = os.getenv("MODEL_SERVICE_ENDPOINT",
+                          "http://0.0.0.0:8001/v1")
+
 parser = argparse.ArgumentParser()
 parser.add_argument("-c", "--chunk_size", default=150)
 parser.add_argument("-e", "--embedding_model", default="BAAI/bge-base-en-v1.5")
 parser.add_argument("-H", "--vdb_host", default="0.0.0.0")
 parser.add_argument("-p", "--vdb_port", default="8000")
 parser.add_argument("-n", "--name", default="test_collection")
-parser.add_argument("-m", "--model_url", default="http://0.0.0.0:8001/v1")
+parser.add_argument("-m", "--model_url", default=model_service)
 args = parser.parse_args()
 
+client = HttpClient(host=args.vdb_host,
```

Comment:
If we are going to move it this far up can we rename it to `vectorDB_client`?

Reply:
Sure, i moved that there bc i was getting an error "client is not defined in line 32"

Reply:
Yeah, I figured it must have been throwing an error. Strange I never noticed it. Thanks :)